### DISCUSSION TITLE
application wishes

### DISCUSSION THREAD

**Discussion by chipset2011** [zepp-health/zeppos-forum#61] [User Feedback]
I have a couple of comments about the zeppOS apps, the first thing I would like to have is the ability to enable a new timer in the stopwatch app with the bottom button, like in real stopwatches. It's not as convenient to press on the screen.
Also, once in the bracelet xiaomi mi band2 was a handy alarm clock, which could not be turned off (you can postpone) until you get up and start moving, I would like to have such a mode in the alarm clock app. Device - GTR4

**Comment by MyoungXUE** [comment]
Hello, received two suggestions, both of which I think are very good features.

I forwarded them to the relevant colleagues and will inform you of any new developments.

**Reply by chipset2011** [reply]
I would also like to be able to mute the phone when there is an incoming call from the watch without resetting the call
